### PR TITLE
Hauki 103 integrate cypress

### PR DESCRIPTION
**Pull request by VilleMiekkoja**

### CODE REVIEW

File: cypress/integration/examples/actions.spec.js [hunk 1]
```diff
@@ -0,0 +1,299 @@
+/// <reference types="cypress" />
```
Comment (minevala) [inline cypress/integration/examples/actions.spec.js:1]: What convention should we follow with the generated examples files? 🤔 
Should we keep them or replace them with a readme-link?

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,10 @@
 
 # testing
 /coverage
+/cypress/videos/*
+!/cypress/videos/.gitkeep
```
Comment (minevala) [inline .gitignore:11]: I'm seeing this as deleted file after running tests 🤔

File: cypress/integration/examples/actions.spec.js (hunk 1 above)
Comment (VilleMiekkoja) [reply cypress/integration/examples/actions.spec.js:1]: Yeah readme-link is probably better, gonna add one and delete these examples.

- [ ] Delete examples and provide link to them.

File: .gitignore (hunk 2 above)
Comment (VilleMiekkoja) [reply .gitignore:11]: Should be fixed now

- [ ] Fix cypress deleting .gitkeep

**Review by minevala — APPROVED** [review]
Good job

**Pull request merged**